pyflow/scene/from_ipynb_conversion.py

```diff
@@ -202,24 +197,32 @@ def get_edges_data(blocks_data: OrderedDict) -> OrderedDict:
     return edges_data
 
 
-def get_input_socket_data(socket_id: int) -> OrderedDict:
-    """Returns the input socket's data with the corresponding id."""
+def get_input_socket_data(socket_id: int, block_data: OrderedDict) -> OrderedDict:
+    """Returns the input socket's data with the corresponding id
+    and at the correct relative position with respect to the block."""
+
+    block_width = block_data["width"]
+
     return {
         "id": socket_id,
         "type": "input",
-        "position": [0.0, SOCKET_HEIGHT],
+        "position": [block_width / 2, 0],
```

Comment:
This should depend of the number of sockets in

Reply:
When converting, blocks always have one socket in, don't they?

Reply:
You are right, this will be refactor when we will build the reversibility of a .ipyg -> .ipynb transform.